src/views/Beheer/contactverzoek-formulieren/ContactverzoekFormulierBeheer.vue

```diff
@@ -0,0 +1,614 @@
+<template>
+  <prompt-modal
+    :dialog="cancelDialog"
+    message="Weet je zeker dat je het proces wil annuleren? Niet opgeslagen gegevens zullen verloren gaan."
+    cancel-message="Nee"
+    confirm-message="Ja"
+  />
+
+  <utrecht-heading :level="1">{{
+    props.id ? "Formulier bewerken" : "Formulier aanmaken"
+  }}</utrecht-heading>
+  <template v-if="loading"> <simple-spinner /></template>
+  <div>
+    <pre>
+      <div style="white-space: pre-line">
+      Hier kan je een template maken voor een contactverzoek. Houd er rekening mee dat
+      dit template een aanvulling is op de standaard vragen.
+      Deze hoef je hier dus niet toe te voegen. De standaardvragen zijn: 
+      - Klantnaam
+      - Telefoonnummer 1
+      - Telefoonnummer 2 
+      - Omschrijving telefoonnummer 2 
+      - E-mailadres 
+      - Interne toelichting voor medewerker
+      </div>
+    </pre>
+  </div>
+  <form class="container" @submit.prevent="submit">
+    <label class="utrecht-form-label" for="titel"
+      ><span>Titel *</span>
+      <input
+        class="utrecht-textbox utrecht-textbox--html-input"
+        required
+        type="text"
+        id="titel"
+        v-model="title"
+      />
+    </label>
+    <!-- dropdown for afdelingen -->
+    <label class="utrecht-form-label">
+      <span>Afdeling *</span>
+      <select
+        class="utrecht-select utrecht-select--html-select"
+        v-model="selectedAfdeling"
+        required
+      >
+        <option disabled value="">Kies een afdeling</option>
+        <option
+          v-for="afdeling in afdelingen"
+          :key="afdeling.id"
+          :value="afdeling.naam"
+        >
+          {{ afdeling.naam }}
+        </option>
+      </select>
+    </label>
+
+    <!-- Loop through vragen and render label and input field -->
+    <template v-for="(vraag, index) in vragen" :key="index">
+      <div class="question-container">
+        <div class="question-container-label utrecht-form-label">
+          <span
+            >Vraag {{ index + 1 }} -
+            {{ setVraagTypeDescription(vraag.questiontype) }}</span
+          >
+        </div>
+        <div class="question-container-button">
+          <!-- delete button -->
+          <utrecht-button
+            appearance="secondary-action-button"
+            class="icon icon-after trash icon-only"
+            type="button"
+            @click="removeVraag(vraag.id)"
+          />
+        </div>
+
+        <!-- Conditional part for Input -->
+        <div
+          style="width: 100%"
```

Comment:
geen inline styling svp, in verband met csp (content security policy) restricties

Reply:
er staat er nog een  <div style="white-space: pre-line">

Reply:
die kan ik niet vinden?

Reply:
` <div style="white-space: pre-line">`

Reply:
aangepast